dbx_Pg.pl

```diff
@@ -56,6 +56,47 @@
 				"CREATE INDEX rptrecord_idx_serial6 ON rptrecord (serial, ip6);",
 				],
 			},
+		"tls_report" => {
+			column_definitions 		=> [
+				"serial"					, "bigint"									, "GENERATED ALWAYS AS IDENTITY",
+				"mindate"				, "timestamp without time zone"		, "NOT NULL",
+				"maxdate"				, "timestamp without time zone"		, "NULL",
+				"domain"					, "character varying(255)"				, "NULL",
+				"org"						, "character varying(255)"				, "NOT NULL",
+				"reportid"				, "character varying(255)"				, "NOT NULL",
+				"contact"				, "character varying(255)"				, "NULL",
+				"policy_type"			, "character varying(255)"				, "NULL",
+				"policy_string"		, "character varying(255)"				, "NULL",
+				"summary_failure"		, "bigint"									, "GENERATED ALWAYS AS IDENTITY",
+				"summary_successful"	, "bigint"									, "GENERATED ALWAYS AS IDENTITY",
+				"raw_json"				, "text"										, "",
+				],
+			additional_definitions 		=> "PRIMARY KEY (serial), UNIQUE KEY domain (domain,reportid)",
+			table_options			=> "ROW_FORMAT=COMPRESSED",
+			indexes				=> [],
+				"CREATE UNIQUE INDEX tls_report_uidx_domain ON tls_report (domain, reportid);"
+			},
+			"tls_rptrecord" => {
+			column_definitions 		=> [
+				"id"								, "bigint"							, "GENERATED ALWAYS AS IDENTITY",
+				"serial"							, "bigint"							, "NOT NULL",
+				"result_type"					, "character varying(255)"		, "",
+				"sending_mta_ip"				, "bigint"							, "",
+				"sending_mta_ip6"				, "bytea"							, "",
+				"receiving_mx_hostname"		, "character varying(255)"		, "",
+				"receiving_mx_helo"			, "character varying(255)"		, "",
+				"receiving_ip"					, "bigint"							, "",
+				"receiving_ip6"				, "bytea"							, "",
+				"failed_session_count"		, "bigint"							, "NOT NULL",
+				"additional_information"	, "character varying(255)"		, "",
+				"failure_reason_code"		, "character varying(255)"		, "",
+				],
+			additional_definitions 		=> "KEY serial (serial,sending_mta_ip), KEY serial6 (serial,sending_mta_ip6)",
```

Comment:
Just like above, the index are created by `indexes` on line 96 below. And the definition of primary key is missing:
```suggestion
			additional_definitions 		=> "PRIMARY KEY(id)",
```

Reply:
Thanks for reviewing the code. I knew I was going to leave in stuff I shouldn't.